stl/inc/xstring

```diff
@@ -1678,6 +1701,11 @@ private:
 template <contiguous_iterator _It, sized_sentinel_for<_It> _Se>
 basic_string_view(_It, _Se) -> basic_string_view<iter_value_t<_It>>;
 
+#if _HAS_CXX23

```

Comment:
+1000 for putting the deduction guides in the same order as the constructor declarations despite that the paper failed to do so! Nice catch. (FWIW, I popped over to https://github.com/cplusplus/draft/pull/4676 to ensure the Editors applied the same ordering fix when merging the paper into the Working Draft and discovered they have already done so.)

No change requested.